package.json

```diff
@@ -28,6 +28,7 @@
     "lint": "eslint src/ src-test/ --max-warnings 0",
     "test": "npm run precheck && mocha src-test/ --timeout 60000 --reporter dot && npm run lint"
   },
+  "overrides": { "tough-cookie": "~4.1.3" },
```

Comment:
Is it wise to put an `overrides` in a package? This seems like people should do on a per-project basis, not me as a package maintainer

Reply:
i.e. Thoughts on making this as a workaround explained in #93 until `get-pixels` puts out a proper release?

Reply:
Thats a fair workaround. Definitely your call as the package maintainer - removed the override.

Reply:
Cool, thanks! I'll post a comment to #93 with attribution to you then =)